### ISSUE TITLE
Budgets per milestone per team

### ISSUE THREAD

**Issue by michielbdejong** [federatedbookkeeping/task-tracking#17]
Let's see if we can rewrite the scopes, but keeping the milestone/team budget divisions as they are. In euro terms:

1: PS: 7312.50, m-ld: 6337.50, ED: 0, total: 13650 (27%)
2: PS: 0, m-ld: 7312.5, ED: 10237.5, total: 17550 (35%)
3/4: PS: 7962.50 m-ld: 1625, ED: 5037.50, total: 14625 (29%)
5: PS: 0, m-ld: 3900, ED: 0, total: 3900 (8%)
Total: PS: 15275 (31%), m-ld: 19175 (38%), ED: 15275 (31%)
Total for project: (1) 13650 + (2) 17550 + (3/4) 14625 + (5) 3900 = (PS) 15275 + (m-ld) 15275 + (ED) 19175 = 49725

**Comment by mcalligator** [comment]
Agreed in principle; this is the fairest way forward.  Milestones to be rearranged and rescoped as necessary, albeit without compromising the essence of what the project is intended to deliver.

**Comment by michielbdejong** [comment]
Looking at https://nlnet.nl/rfp/overview?project=2022-10-122 now, it's:
* 1a: 10413
* 1b: 975
* 1c: 2700
* 2a: 7313 (claimed by m-ld)
* 2b: 10238 (deprecated)
* 3a: 5813
* 3b: 1538
* 3c: 1513
* 3d: 2326
* 3e: 3000
* 4a: 1950
* 4b: 1940

Total: 32718 left + 7313 claimed + 10238 deprecated = 49729
